perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/widgets/Select.java

```diff
@@ -59,12 +56,25 @@
 	private static final String DESELECT_ALL = "deselectAll";
 	private static final String TRUE = "true";
 
-	private final AttributeMixin<Select> attributeMixin = new AttributeMixin<Select>(this);
-	private final FocusableMixin<Select> focusableMixin = new FocusableMixin<Select>(this);
-	private final EnabledMixin<Select> enabledMixin = new EnabledMixin<Select>(this);
+	private static final String BOOTSTRAP_SELECT = "bootstrap-select";
```

Comment:
This "Select" implementation works, thank you. But the problem is at ``cz.metacentrum.perun.wui.registrar.widgets.items.ValidatedEmail`` where in ``onAttach()`` method we reference HTML elements directly and append classname "form-control". Please check this method and fix calls to get proper element since tree structure probably changed. ``<Button>`` element should be the one with appended classname.